### ISSUE TITLE
Translate to python

### ISSUE THREAD

**Issue by florianboergel** [hdrake/simplEarth#9]
I am still impressed with the content of this lecture series. I offer to translate everything (climate related) to python. I get that this course is mainly to push Julia, therefore the translation could be perceived as counterproductive. 

But I guess my main point is, that language should not be a barrier to this amazing content.

**Comment by hdrake** [comment]
Thanks for your interest! What do you think is the best way of organizing this? Should we make a simplEarth organization and have `simplEarth.py` and `simplEarth.jl`?

**Comment by hdrake** [comment]
E.g. one reason to separate them is that it makes the environment set-up faster, which is especially important for binder.

**Comment by fonsp** [comment]
I agree that a python port would benefit everyone, but remember that there is no python equivalent of Pluto.jl yet, so you need to find alternatives for the interactivity

**Comment by florianboergel** [comment]
I agree. It's a good idea to separate both versions. I propose that I start by creating simplEarth.py

@fonsp to your point: Have a look at 
https://florianboergel.github.io/personal_blog/jupyter/2020/11/12/nonlinear-dynamics.html

Interactivity of ipywidgets is limited. However I would be OK with the way it is handled here. Try the binder version. 

The simplEarth.py could be hosted similar to [this](https://florianboergel.github.io/personal_blog/) blog version on the MIT class github page?

**Comment by florianboergel** [comment]
Quick update: I finished with the two-dimensional advection-diffusion of heat. The last lecture is exciting but I am not so much looking forward to translating it to python ;). The speed difference between python and julia is just mindblowing.
